src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/SwaggerGenServiceCollectionExtensions.cs

```diff
@@ -33,8 +32,16 @@ public static class SwaggerGenServiceCollectionExtensions
             services.TryAddTransient<ISerializerDataContractResolver>(s =>
             {
 #if (!NETSTANDARD2_0)
-                var serializerOptions = s.GetService<IOptions<JsonOptions>>()?.Value?.JsonSerializerOptions
+                var serializerOptions =
```

Comment:
It might be a good idea to document this order of options somewhere, because not documenting it will most likely raise questions about what options are used.